src/index.html

```diff
@@ -5,9 +5,299 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="\images\favicon.png"
+    type="image/png"
+    rel="shortcut icon">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__top">
+        <img
+          class="header__logo logo"
+          src="images/logo.png"
+          alt="Miami condo kings logo"
+        >
+        <div class="header__phone">
+        +1 234 555-55-55
+        </div>
+        <a href="tel:+1 234 555-55-55" class="header__call nav__link">
+          call to order
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item--active">
+              Home
+          </li>
+          <li class="nav__item">
+            <a href="#about" class="nav__link">
+              About us
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gallery" class="nav__link">
+              Gallery
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#consultation" class="nav__link">
+              Consultation
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#testimonials" class="nav__link">
+              Testimonials
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#contacts" class="nav__link">
+              Contacts
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="section section__first" id="about">
+        <div class="info">
+          <h1 class="info__heading">Miami <strong>best</strong><br>Real estate <br><a href="#" class="link">here</a></h1>
+          <div class="info__text">
+            <p>
+              <strong>50 years experience in love.</strong>
+            </p>
+            <p>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
```

Comment:
```suggestion
            <p>
              Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
            </p>
```
Переносим не только закрывающий тег, но и контент после открывающего